schema.sql

```diff
@@ -57,4 +57,36 @@ ALTER TABLE animals ADD COLUMN species_id INT references species(id);
 /* Add 'owner_id' column to animals table with foreign
    key referencing 'owners' table
 */
-ALTER TABLE animals ADD COLUMN owner_id INT references owners(id);
\ No newline at end of file
+ALTER TABLE animals ADD COLUMN owner_id INT references owners(id);
+
+/* Creating 'vets' table in 'vet_clinic' database
+   Table has columns: id, name, age, and date_of_graduation
+*/
+CREATE TABLE vets (
+    id  BIGSERIAL  NOT NULL,
+    name   TEXT NOT NULL,
+    age INT,
+    date_of_graduation DATE,
+    PRIMARY KEY(id)
+);
+
+/* Creating 'specialties' table in 'vet_clinic' database
+   Table has columns: id, specie, and vet
+*/
+CREATE TABLE specialties (
+    id  BIGSERIAL  NOT NULL,
+    vet TEXT,
+    specie TEXT,
+    PRIMARY KEY(id)
+);
```

Comment:
- [ ] Kindly make sure that name of the table is specializations.
- [ ] Add only species_id & vet_id columns inside the table which is of int data type.

Reply:
Hello @sja-thedude! The instruction on the project details does not specify what data type to use for the relation of the `vet` and `specie`.
![Screenshot from 2022-10-20 12-49-01](https://user-images.githubusercontent.com/101870993/196940669-a26baddc-f278-4a2a-82b5-cfa8a17edb78.png)
I choose to use text-based type with names (`vet` & `species`) because it is easier to read than having numbers in form of `id`.

Please can you explain to me why you think using the numbered ID is better?

Reply:
- [ ] I have advised to use it as its a join table and ids are taken to access it.

Reply:
OK! Thanks so much for the clearification.